pages/common/openssl-ts.md

```diff
@@ -0,0 +1,20 @@
+# openssl ts
+
+> OpenSSL command to generate and verify timestamps.
+> More information: <https://www.openssl.org/docs/manmaster/man1/openssl-ts.html>.
+
+- Generate sha512 timestamp request of `file` (output to `file.tsq`):
+
+`openssl ts -query -data {{file}}  -sha512  -out {{file.tsq}}`
+
+- Check timestamp response file `file.tsr`:
```

Comment:
I don't like this sentence @bl-ue Any suggestions?

Reply:
How about this?
- Check date and metadata of timestamp response file `file.tsr`:

Reply:
I like that you are inputing, my problem is the `file file` bit

Reply:
Lol, I completely missed that.

Reply:
- Check date and metadata from `file.tsr`, the timestamp response file:

How about this?

Reply:
Well I haven't used this tool, so I don't understand the context but that doesn't read right to me

- Check the date and metadata from a specified timestamp response file

Reply:
Are you sure it's a good idea not to reference that the timestamp response file is the argument with the .tsr extension?

You submit the timestamp request file (.tsq) to the time stamp server, it returns a time stamp request file (.tsr). This shows the date on the tsr and the metadata, but it doesn't verify the server's signature.

Here is a sample output for the command:

Using configuration from /usr/lib/ssl/openssl.cnf
Status info:
Status: Granted.
Status description: unspecified
Failure info: unspecified

TST info:
Version: 1
Policy OID: tsa_policy1
Hash Algorithm: sha512
Message data:
    0000 - cf 83 e1 35 7e ef b8 bd-f1 54 28 50 d6 6d 80 07   ...5~....T(P.m..
    0010 - d6 20 e4 05 0b 57 15 dc-83 f4 a9 21 d3 6c e9 ce   . ...W.....!.l..
    0020 - 47 d0 d1 3c 5d 85 f2 b0-ff 83 18 d2 87 7e ec 2f   G..<]........~./
    0030 - 63 b9 31 bd 47 41 7a 81-a5 38 32 7a f9 27 da 3e   c.1.GAz..82z.'.>
Serial number: 0x9826A4
Time stamp: Jun 14 21:59:39 2021 GMT
Accuracy: unspecified
Ordering: yes
Nonce: unspecified
TSA: DirName:/O=Free TSA/OU=TSA/description=This certificate digitally signs documents and time stamp requests made using the freetsa.org online services/CN=www.freetsa.org/emailAddress=busilezas@gmail.com/L=Wuerzburg/C=DE/ST=Bayern
Extensions:

Reply:
My suggestion uses it, and the file extension specifies it in the command

Reply:
Ok, I implemented your suggestion.